daemon/openrazer_daemon/dbus_services/dbus_methods/basiliskv3.py

```diff
@@ -0,0 +1,97 @@
+from openrazer_daemon.dbus_services import endpoint
+
+
+@endpoint('razer.device.scroll', 'setScrollMode', in_sig='y')
+def set_scroll_mode_basilisk_v3(self, mode):
```

Comment:
remove the `basilisk_v3` suffix from all the methods here, no point in doing yet (I'm aware that existing methods have that naming scheme).

Also the file name basiliskv3.py is not great, try to use a generic name or put it in a different existing file

Reply:
When it comes to the module name, went for `mouse_scroll_wheel` as it seemed to be at the best level of generic for me. Let me know if that's acceptable.